streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -668,8 +668,7 @@ public final <KIn, VIn> void addGlobalStore(final String sourceName,
                                      topic,
                                      processorName,
                                      stateUpdateSupplier,
-                                     storeFactory.storeName(),
-                                     storeFactory.loggingEnabled());
+                                     storeFactory.storeName());
```

Comment:
Change this to not fail if state-store-logging is enabled, but instead just disable it. This align to what we do for "read-only stores".

Noticed the difference between read-only vs global and though it better to align both.